illumidesk/templates/jupyter_ingress.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if .Values.illumidesk.enabled }}
```

Comment:
Added same albIngressController conditional and updted jupyter_ingress.yaml to alb_ingress.yaml. 
By default helm installs deployments before ingress manifests but I have noted this down in case we need this in the future. Here is the link for order of manifests installation.
https://github.com/helm/helm/blob/release-2.10/pkg/tiller/kind_sorter.go#L29